diagnostic.md

```diff
@@ -53,9 +69,23 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
-<!-- your answer starts here -->
+```CSS
+.big {
+  ...
+}
 
-<!-- your answer ends here -->
+#contentPane {
+  ...
+}
+
+.important .red {
+  ...
+}
+
+div p {
```

Comment:
I actually did that at first and then changed it at the last second. Should have gone with my gut. Hadn't thought of the difference between child and sibling selectors before. Good to know.